pennylane/measurements/expval.py

```diff
@@ -119,7 +130,11 @@ def process_state(self, state: Sequence[complex], wire_order: Wires):
             idx = int("".join(str(i) for i in self.obs.parameters[0]), 2)
             probs = qml.probs(wires=self.wires).process_state(state=state, wire_order=wire_order)
             return probs[idx]
-        eigvals = qml.math.asarray(self.obs.eigvals(), dtype="float64")
+        eigvals = (
+            qml.math.asarray(self.obs.eigvals(), dtype="float64")
+            if not isinstance(self.obs, MeasurementValue)
+            else qml.math.asarray(qml.math.arange(0, 2 ** len(self.wires)), dtype="float64")
```

Comment:
Yes, because I can't really control whether or not someone combines multiple measurement values using arithmetic operators, but I can document that doing that will result in unexpected/incorrect results. The next PR will add multi-measurement support too

Reply:
Ah, okay, I had misunderstood, I thought only single measurement was going to be supported. Thanks!